src/components/HighlightedMarkdown/H.js

```diff
@@ -16,12 +24,12 @@ export default class H extends PureComponent {
     children: PropTypes.node.isRequired
   }
 
-  onClick = () => {
+  onClick() {
     // We need to set the hash manually because <Link/> will not.
     // Also we can't use `onSetActive` callback because it doesn't work if
     // an element is on very bottom so that there is no scroll possible.
     setTimeout(() => {
-      location.hash = this.props.children[0].props.to
+      location.hash = this.to
```

Comment:
seems to be a property on the DOM element - personally I recommend the approach you guys were doing before, using the class-bound arrow methods and reaching into props.  VDOM scales a lot more nicely when the DOM is treated as write-only.